specification/gedcom-03-datamodel.md

```diff
@@ -2971,6 +2971,7 @@ are not currently given their own tag. Cultural or personal gender preference ma
 | `CHALLENGED` | Linking this child to this family is suspect, but the linkage has been neither proven nor disproven. |
 | `DISPROVEN` | There has been a claim by some that this child belongs to this family, but the linkage has been disproven. |
 | `PROVEN` | There has been a claim by some that this child does not belong to this family, but the linkage has been proven. |
+| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. `UNCHALLENGED` is implied if this structure is not present. |
```

Comment:
```suggestion
| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. |
```

Remove the "if not present" clause due to comments in pull request conversation thread